tests/complex/module_collision/reference

```diff
@@ -1,8 +1,8 @@
 [[1m[31mError[0m]: Local module conflict,
- Module [1mA[0m is provided simultaneously by [1mmodule_collision/Z/a.mli[0m and
-[1mmodule_collision/B/A.mli[0m
+ Module [1mA[0m is provided simultaneously by [1mmodule_collision/B/A.mli[0m and
+[1mmodule_collision/Z/a.mli[0m
```

Comment:
Did you run this test before your changes? It looks like a possibly unrelated change.

Reply:
Ah yes, you are right, the error appeared before, I will split the commit in two.